src/ray/protobuf/autoscaler.proto

```diff
@@ -0,0 +1,160 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+package ray.rpc;
+
+message ResourceBundle {
+  // The fixed point resource size of the resource (granularity: 1/10000).
+  map<string, int64> resources = 1;
+}
+
+// A workload (task/actor/placement group) unfulfilled by the the cluster's internal scheduler(s).
+message QueuedResources {
```

Comment:
Renamed `ResourceRequest` to this since "request" is too confusing/repetitive with grpc terminology.